### ISSUE TITLE
Unable to use custom docker network (Error P1000)

### ISSUE THREAD

**Issue by kmanwar89** [Peppermint-Lab/peppermint#117]
I've spent some time troubleshooting, trying to figure out what this issue is, but I'm pulling my hair out here.

The issue I'm having is I cannot get Peppermint to work when I add in my custom docker network - if I just use the default network that docker-compose creates, there's no issues. However, if I add my custom network into my compose file and change nothing else, it fails to load. 

Here is my working docker-compose file. I had to adjust the port numbers to 5001 per issue #80. 

```
  GNU nano 6.2                                                                                                    docker-compose.yml *                                                                                                           
version: "3.1"

services:
  postgres:
    container_name: postgres
    image: postgres:latest
    restart: always
    volumes:
      - ./docker-data/db:/data/db
    environment:
      POSTGRES_USER: peppermint
      POSTGRES_PASSWORD: 1234
      POSTGRES_DB: peppermint

  client:
    container_name: peppermint
    image: pepperlabs/peppermint:latest
    ports:
      - 5001:5001
    restart: on-failure
    depends_on:
      - postgres
    environment:
      PORT: 5001
      DB_USERNAME: peppermint
      DB_PASSWORD: 1234
      DB_HOST: 'postgres'
      BASE_URL: "http://192.168.1.10:5001"
```

I then add the stanza for my custom network and re-create the container by issuing _docker compose up_.  Here is what I see in the output of the container logs:

```
❯ docker container logs peppermint
yarn run v1.22.19
$ prisma migrate deploy
Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Datasource "db": PostgreSQL database "peppermint", schema "public" at "postgres:5432"

Error: P1000: Authentication failed against database server at `postgres`, the provided database credentials for `peppermint` are not valid.

Please make sure to provide valid database credentials for the database server at `postgres`.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
```

I'd appreciate any help with solving this, thanks!

**Comment by potts99** [comment]
perhaps this could be a hostname issue for the DB_HOST?

**Comment by kmanwar89** [comment]
> perhaps this could be a hostname issue for the DB_HOST?

Thank you! I am not entirely sure *why* it was erroring before, but I managed to fix my issue (for now, at least).

I changed my container name to be postgres_peppermint, to help distinguish it from other postgres containers I may have running.  I simply updated the DB_HOST field to include this new name, and it is now working.

I just want to know that I also had to update my BASE_URL to include https://domainname.com for it to work properly with NGINX Proxy Manager.  Hopefully this helps anyone else who comes across this issue, thanks!
